Modules/LearningSequence/classes/class.ilObjLearningSequence.php

```diff
@@ -89,6 +89,16 @@ public function storeLSItems(array $ls_items)
 		$db->storeItems($ls_items);
 	}
 
+	/**
+	 * Delete post conditions for ref ids.
+	 * @param int[]
+	 */
+	public function deletePostConditions(array $ref_ids)
```

Comment:
deletePostConditionsForSubObjects?
in Abgrenzung zu deleteAllPostConditions, wenn die Sequenz gelöscht wird?